app/renderer/components/common/switchControl.js

```diff
@@ -36,29 +38,49 @@ class SwitchControl extends ImmutableComponent {
     >
       {
         this.props.leftl10nId && this.props.topl10nId
-        ? <div className='switchControlText'><div className='switchControlLeftText'><div className='switchSpacer'>&nbsp;</div><label className='switchControlLeftText' onClick={this.onClick} data-l10n-id={this.props.leftl10nId} /></div></div>
+        ? <div className='switchControlText'>
+          <div className='switchControlLeftText'>
+            <div className='switchSpacer'>&nbsp;</div>
+            <label className={cx({
+              [css(styles.switchControlText__label, styles.switchControlText_left__label, this.props.small && styles.switchControlText__label_small)]: true,
```

Comment:
better if
```
[css(styles.switchControlText__label]: true,
[css(styles.switchControlText_left__label): true,
[styles.switchControlText__label_small]: this.props.small
```
lmk if you disagree

Reply:
If you set `background: #000` to `switchControlText__label` and `background: #111` to `switchControlText_left__label`, we get `background: #000` instead of the latter, which is what we want. I prefer readability but we have to assure that the styles are surely cascaded is preferred. 

For a while I tried another way to set them like this

````
[css(
  styles.switchControlText__label,
  styles.switchControlText_left__label
)]
````

but it was avoided because Standard complained about that the `[` and `]` should appear on the very same line.

Reply:
Still I believe this is a kind of issue on the transitional phase until you would remove `cx` by creating components in a way which @petemill has demonstrated.

We could create an issue to track the work. It seems to me that replacing `customLeftTextClassName` with `this.props.custom` as you @cezaraugusto did on `browserButton` should work. I took that way for `commonForm` on #10978, e.g. `<CommonForm custom={styles.commonForm_small}>`.